llvm/test/CodeGen/AArch64/GlobalISel/form-bitfield-extract-from-and.mir

```diff
@@ -6,6 +6,10 @@
 #
 # and (lshr x, cst), mask -> ubfx x, cst, width
 
+# LSB = 5
```

Comment:
Please precommit all the formatting changes in this files. I can't tell if there are any changes related to the patch.

Reply:
The update script deleted the comments and I had to move them up to #.

Reply:
I understand that, but please commit all those formatting changes as a separate NFC patch.